Dockerfile

```diff
@@ -6,6 +6,7 @@ ENV GO111MODULE=on
 WORKDIR /src
 COPY . .
 
+RUN make clean
```

Comment:
I think `make clean` is not need here.

Reply:
I added this to not copy the local bin when testing with a single binary.